File: dotnet/turnos/Models/Db/Paciente.cs
namespace turnos.Models.Db;

public class Paciente
{
    public int PacienteId { get; set; }
    public string Apellidos { get; set; } = string.Empty;
    public string Nombres { get; set; } = string.Empty;
    public int DNI { get; set; }
    public DateTime FechaDeNacimiento { get; set; }
    public char Sexo { get; set; }
    public int ObraSocialId { get; set; }
    public ObraSocial ObraSocial { get; set; } = null!;
    public string Credencial { get; set; } = string.Empty;
    public int UsuarioId { get; set; }
    public Usuario Usuario { get; set; } = null!;
    public bool EsUsuario { get; set; }
}

File: parte2/web/src/app/modelos/paciente.ts
export interface Paciente {
    id?: number;
    nombres: string;
    apellidos: string;
    dni: string;
    fechaDeNacimiento: string;
    obraSocialId?: number;
    sexo?: 'M' | 'F' | 'X';
    credencial?: string;
    esUsuario?: boolean;
}